dotnetv3/Keyspaces/Tests/KeyspacesWrapperTests.cs

```diff
@@ -0,0 +1,243 @@
+        {
+            _configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("testsettings.json") // Load test settings from .json file.
+                .AddJsonFile("testsettings.local.json",
+                    true) // Optionally load local settings.
+                .Build();
+
+            _client = new AmazonKeyspacesClient();
+            _wrapper = new KeyspacesWrapper(_client);
+
+            _keyspaceName = _configuration["KeyspaceName"];
+            _tableName = _configuration["TableName"];
+
+            // Define the schema for the test table.
+            var allColumns = new List<ColumnDefinition>
+            {
+                new ColumnDefinition { Name = "title", Type = "text" },
+                new ColumnDefinition { Name = "year", Type = "int" },
+                new ColumnDefinition { Name = "release_date", Type = "timestamp" },
+                new ColumnDefinition { Name = "plot", Type = "text" },
+            };
+
+            var partitionKeys = new List<PartitionKey>
+            {
+                new PartitionKey { Name = "year", },
+                new PartitionKey { Name = "title" },
+            };
+
+            _schemaDefinition = new SchemaDefinition
+            {
+                AllColumns = allColumns,
+                PartitionKeys = partitionKeys,
+            };
+        }
+
+        /// <summary>
+        /// Test the call to CreateKeyspace. This test waits 10 seconds
+        /// for the keyspace to be created.
+        /// </summary>
+        /// <returns>An async Task.</returns>
+        [Fact()]
+        [Order(1)]
+        [Trait("Category", "Integration")]
+        public async Task CreateKeyspaceTest()
+        {
+            _keyspaceArn = await _wrapper.CreateKeyspace(_keyspaceName);
+
+            // Wait until keyspace has been created.
+            Thread.Sleep(10000);
+            var getKeyspaceArn = "";
+            do
+            {
+                getKeyspaceArn = await _wrapper.GetKeyspace(_keyspaceName);
+            } while (getKeyspaceArn != _keyspaceArn);
+            Assert.NotNull(_keyspaceArn);
+        }
+
+        /// <summary>
+        /// Tests the ability to get information about the new keyspace.
+        /// </summary>
+        /// <returns>An async Task.</returns>
+        [Fact()]
+        [Order(2)]
+        [Trait("Category", "Integration")]
+        public async Task GetKeyspaceTest()
+        {
+            var keyspaceArn = await _wrapper.GetKeyspace(_keyspaceName);
+            Assert.Equal(keyspaceArn, _keyspaceArn);
+        }
+
+        /// <summary>
+        /// Tests the CreateTable method. It was necessary to add a
+        /// try/catch to this method since, if the table hasn't been
+        /// created yet, it raises a resource not found error. Without
+        /// the wait and the try/catch, the rest of the tests will
+        /// always fail as well.
```

Comment:
Updates to second and third sentences: 
It's necessary to add a try/catch to this method because if the table hasn't been created yet, it raises a resource not found error. Without the wait and the try/catch, the rest of the tests will always fail, too.